src/bun.js/node/path.zig

```diff
@@ -2905,9 +2907,10 @@ pub fn toNamespacedPathWindowsJS_T(comptime T: type, globalObject: *jsc.JSGlobal
 pub fn toNamespacedPathJS_T(comptime T: type, globalObject: *jsc.JSGlobalObject, allocator: std.mem.Allocator, isWindows: bool, path: []const T) bun.JSError!jsc.JSValue {
     if (!isWindows or path.len == 0) return bun.String.createUTF8ForJS(globalObject, path);
     const bufLen = @max(path.len, PATH_SIZE(T));
-    const buf = try allocator.alloc(T, bufLen);
+    // +8 for possible UNC prefix, +1 for null terminator
+    const buf = try allocator.alloc(T, bufLen + 8 + 1);
```

Comment:
Technically only needs to be `bufLen + 6 + 1` because `toNamespacedPathWindowsT` overwrites the first two bytes of the path. But it's bad that we have to keep track of the required buffer size in multiple places.

Reply:
agree